previewer_jupyter/indico_previewer_jupyter/controllers.py

```diff
@@ -45,9 +47,9 @@ def _process(self):
 
         response = current_app.response_class(html)
         # Use CSP to restrict access to possibly malicious scripts or inline JS
-        csp_header = f"script-src cdn.mathjax.org 'nonce-{nonce}';"
-        response.headers['Content-Security-Policy'] = csp_header
-        response.headers['X-Webkit-CSP'] = csp_header
+        # csp_header = f"script-src cdnjs.cloudflare.com cdn.jsdelivr.net 'nonce-{nonce}';"
+        # response.headers['Content-Security-Policy'] = csp_header
+        # response.headers['X-Webkit-CSP'] = csp_header
```

Comment:
We should keep the CSP; it's needed to reduce the XSS risk from previewed notebooks